web_src/js/components/RepoBranchTagSelector.vue

```diff
@@ -123,6 +123,7 @@ const sfc = {
       return -1;
     },
     scrollToActive() {
+      // eslint-disable-next-line no-jquery/variable-pattern
       let el = this.$refs[`listItem${this.active}`];
```

Comment:
```suggestion
      let el = this.$refs[`listItem${this.active}`]; // eslint-disable-line no-jquery/variable-pattern
```

In general I prefer this form if there is enough space on the line.